### PR TITLE
Adds debug classes

### PR DESCRIPTION
**Pull request by kristoferjoseph**

### CODE REVIEW

File: debug.mjs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+.debug * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * * * *:hover { border:2px solid var(--debug-color, darkviolet) }
+.debug * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+.debug * * * * * * * * *:hover { border:2px dotted var(--debug-color, darkviolet) }
+  `
+}
```
Comment (colepeters) [inline debug.mjs:21]: Ahhh shoot, is this because of Firefox missing support for `has`?

Comment (kristoferjoseph) [reply debug.mjs:21]: Yes. I think I'll try it with is() to slim down the selector though. Then when Firefox 116 drops I'll move it to use has as well

Comment (colepeters) [reply debug.mjs:21]: Sounds good!

Comment (kristoferjoseph) [reply debug.mjs:21]: `is()` didn't help.

Comment (colepeters) [reply debug.mjs:21]: oh noooo 😂

Comment (colepeters) [reply debug.mjs:21]: However, I don't think the extra `*` selectors are needed? `.debug *` should do it, peep this: https://codesandbox.io/s/cranky-darkness-spxmdz?file=/styles.css

Comment (kristoferjoseph) [reply debug.mjs:21]: Correct. I had different styling for the different levels but not needed now.

Comment (colepeters) [reply debug.mjs:7]: Ohhhh indeed, I totally missed that

**Review by colepeters — APPROVED** [review]
😍

**Pull request merged**